prow/repoowners/repoowners.go

```diff
@@ -394,7 +394,10 @@ func (c *Client) cacheEntryFor(org, repo, base, cloneRef, fullName, sha string,
 			log.WithField("duration", time.Since(start).String()).Debugf("Completed loadAliasesFrom(%s, log)", gitRepo.Directory())
 
 			start = time.Now()
-			ignoreDirPatterns := c.ownersDirBlacklist().ListIgnoredDirs(org, repo)
+			var ignoreDirPatterns []string
+			if ownersDirDenylist := c.ownersDirDenylist(); ownersDirDenylist != nil {
```

Comment:
this is still broken for me, it looks like this change was deployed in the latest prow image already?

In my PR, the bot is detecting OWNERS files in my vendor dir: 
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/792

the type of OwnersDirBlacklist changed from `OwnersDirBlacklist`  to `pointer to OwnersDirDenylist`.

Previously if ownersDirBlacklist() returned an empty OwnersDirBlacklist struct, then ownersDirBlacklist.ListIgnoredDirs would still add the default ignored dirs.

Now if ownersDirDenylist is nil, the ownersDirBlacklist.ListIgnoredDirs don't get added.